src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im2.contexts.onboarding.identifiers.profile-card.style
+  (:require [react-native.reanimated :as reanimated]
+            [quo2.foundations.colors :as colors]))
+
+(def card-header
+  {:flex-direction  :row
+   :justify-content :space-between})
+
+(defn opacity
+  [opacity]
+  {:style (reanimated/apply-animations-to-style
```

Comment:
Should not set style prop here, just define the style

Reply:
Fixed, thanks @briansztamfater